database/ioc/src/Makefile

```diff
@@ -1,4 +1,4 @@
-TOP=../..
+TOP=../../..
```

Comment:
There should be an entry in the `database/configure/RELEASE` file pointing to `$(TOP)/..`, which is how this is handled in the other embedded tops in this module. My guess is that when he worked on this code, Marty's development workspace had a `RELEASE.local` file in the parent V4 directory that included a pointer to the pvExampleTestCPP directory, since that would also provide a way for the build to find that powerSupply stuff from `pvExampleTestCPP/src` and would explain why he never saw the problem.

Given that this module has been superceded by moving its code into the enhanced exampleCPP module we don't really want to spend much time on this, and I've suggested that Marty remove everything except an explanatory README.md file from the master branch to document that. I'd be happy to merge a modified version with your other changes into the release/4.1 branch for users still on the V4.5.0.x series if you think that's worth doing.